docs/app/pages/components/sections/panels/item-display-panel-inline/snippets/app.html

```diff
@@ -3,7 +3,7 @@
 <div class="demo-attributes item-display-panel-inline-container">
     
     <div class="table-responsive preview-pane-table-demo">
-        <table class="table table-hover item-display-panel-inline" [class.animate]="animate" 
+        <table class="table table-hover item-display-panel-inline animate" 
```

Comment:
The panel is currently changing the `left` position. Could we change this to use `transform: translateX` instead as animating left causes the browser to have to perform layout changes each frame, whereas transform will give us a much smoother animation.

Also consider adding a `will-change` property to the inline classes as well. This will allow you to inform the browser to expect certain properties to animate so it can optimize for them